posts/setting-up-a-shared-library-and-seed-job-in-jenkins-1.md

```diff
@@ -0,0 +1,134 @@
+## Introduction
+I have been working at a client the past few months that has adopted a desire for a consistent [JHipster](https://www.jhipster.tech/) microservice architecture. We have more than ten microservices that are supported by multiple teams. Developers support multiple services and we needed a uniform approach to deploy quality code quicly to the cloud. We set up Jenkins to support building and testing any branch of all services while maintaing the ability to deploy releases along side feature development. As Jenkins grew, we decided to maintain our Jenkins related code through source control. 
+
+In order for us to satisfy our requirements, we used a couple of the advanced Jenkins tools.
+  * [Jenkins Job DSL API](https://jenkinsci.github.io/job-dsl-plugin/)
+  * [Jenkins Shared libraries](https://jenkins.io/doc/book/pipeline/shared-libraries/)
+  
+This is the first of a two part series where the target audience is anyone interested in streamlining the process to onboard new services.
+
+### Source Code
+The source code is available below
+  * Jenkins Shared Library ([`microservice-pipelines`](https://github.com/kcrane3576/microservice-pipelines))
```

Comment:
Why is it 'Jenkins Shared Library' in part 1?

Reply:
I introduce it in the first blog to set up the reader to be prepared to use a repository for the code associated with the Jenkins code we plan to use